components/CheckDocumentation/vacuum/MxidWraparound.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const MxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when the multixact ID utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once the utilization exceeds{" "}
+      <code>{critical}%</code>. Resolves once the utilization falls below the
+      threshold.
+    </p>
+  );
+};
+
+const MxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        The multixact ID is close to wraparound. This is about a multixact ID,
+        not about the normal transaction ID.
+      </p>
+      <p>
+        A multixact ID is an internal identifier used support row locking by
+        multiple transactions. Multixact IDs are created when transactions use{" "}
+        <code>SELECT ... FOR UPDATE</code> to lock and update tuples.
```

Comment:
Is this the only case where MXIDs are used? I don't think we have to be exhaustive here, but if these could be coming from other places, we should avoid implying that this is the one source, since users may get confused if they have MXID problems but are _not_ using <code>SELECT ... FOR UPDATE</code>.